java/set-min-max-scale/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.esri.arcgisruntime.samples.setminxmaxscale">
+
+    <uses-permission android:name="android.permission.INTERNET" />
+
+    <uses-feature
+        android:glEsVersion="0x00020000"
```

Comment:
Samples using 3D (scenes) require Open GL ES 3.0. Maps requires 2.0, but uses 3.0 if available. May need to check this on Integrated Mesh sample, as I may have forgotten to highlight it to you.

Reply:
This sample is just 2D.